pkgs/development/python-modules/gdb-pt-dump/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "gdb-pt-dump";
+  version = "0-unstable-2024-04-01";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "martinradev";
+    repo = "gdb-pt-dump";
+    rev = "50227bda0b6332e94027f811a15879588de6d5cb";
```

Comment:
This version is incompatible with pwndbg `2024.02.14`. It will throw import error. 
When using with qemu - https://github.com/martinradev/gdb-pt-dump/issues/29
Use commit `89ea252f6efc5d75eacca16fc17ff8966a389690` 

or create patch like in this PR https://github.com/pwndbg/pwndbg/pull/2097/files#diff-ab70d059e0a6ccf28211f26220d4b4a174143af8b0e4adbbd033f43ed7f72b52R497